README.md

```diff
@@ -1,11 +1,13 @@
 # Deprecated
```

Comment:
I think we can freely remove `Deprecated` tag.

The fact that st2chatops relies on st2client.js makes all the difference.